src/App.test.js

```diff
@@ -28,3 +30,13 @@ it('navigates home when you click the home link', () => {
   // Check correct page content showed up
   expect(document.body.textContent).toBe('Home | Instructions | About');
 });
+
+it('should render a button with the class of button', () => {
+  render(<ButtonComponent />);
+});
+
```

Comment:
here is where we would want to enter an assertion.  in your example you are checking to see that your button has the classname 'button'.  the assertion can be something LIKE 
'expect(<ButtonComponent.>.className).toBe('button')'
my syntax may be off but the point is that the test will fail if your target's classname is not 'button'